contrib/fwupd.spec.in

```diff
@@ -235,6 +235,7 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %if 0%{?enable_dummy}
 %{_libdir}/fwupd-plugins-2/libfu_plugin_test.so
 %endif
+%{_libdir}/fwupd-plugins-2/libfu_plugin_thunderbolt.so
```

Comment:
Can you split the enabling of thunderbolt in CI off to a separate PR please, that needs merging straight away.

Reply:
https://github.com/hughsie/fwupd/pull/212

Reply:
Thanks, merged. If you rebase this PR now I think we're in a position to merge it.

Reply:
Done and passed CI